Automattic/wp-calypso
# Plans: Show trial label instead of plan name in sidebar when current plan is on free trial

This pull request fixes https://github.com/Automattic/wp-calypso/issues/1659 by displaying `Trial` instead of the plan name next to the `Plan` menu item in the sidebar when the current plan is on trial:

![screenshot](https://cloud.githubusercontent.com/assets/594356/12114224/5de9c87a-b3a8-11e5-8d9d-c92ce9dc2475.png)
 
#### Testing instructions

1. Run `git checkout fix/sidebar-plan-label` and start your server
2. Get a new test account and blog
3. Enable your proxy
5. Select a plan by clicking the `Start Free Trial` button on the [`Plans` page](http://calypso.localhost:3000/plans)
6. Click the `Start 14 Day Free Trial` button on the `Secure Payment` page
7. Check that the purchase was successful
8. Navigate to any page of the `My Sites` section
9. Check that `Trial` is displayed in the sidebar
10. Upgrade your plan from the `Plans` page
11. Check that the plan name is now displayed in the sidebar

#### Reviews
